src/index.html

```diff
@@ -22,6 +22,412 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="header">
+        <a
+          href="#home"
+          class="logo-image"
+        >
+          <img
+            src="images/logo.png"
+            alt="logo"
+          />
+        </a>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#Apple"
+                class="nav__link is-active"
+              >
+                Apple
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#Samsung"
+                class="nav__link"
+              >
+                Samsung
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#Smartphones"
+                class="nav__link"
+              >
+                Smartphones
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#Laptops&Computers"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#Gadgets"
+                class="nav__link"
+              >
+                Gadgets
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#Tablets"
+                class="nav__link"
+              >
+                Tablets
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#Photo"
+                class="nav__link"
+              >
+                Photo
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#Video"
+                class="nav__link"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        />
+
+        <article class="card__info card__info--title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </article>
```

Comment:
```suggestion
        <h3 class="card__info card__info--title">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h3>
```
Consider using semantic tags for titles

Reply:
Done